src/app/interaction-model/IterationTypes.h

```diff
@@ -70,8 +70,13 @@ struct AttributeEntry
 /// Iteration rules:
 ///   - kInvalidEndpointId will be returned when iteration ends (or generally kInvalid* for paths)
 ///   - Any internal iteration errors are just logged (callers do not handle iteration CHIP_ERROR)
```

Comment:
This is missing some comments from https://github.com/project-chip/connectedhomeip/pull/32914

Reply:
Updated in #33345

Because that PR also does renaming, it starts being hard to both rename and update comments.